src/main/java/com/epam/test_generator/dto/UserDTO.java

```diff
@@ -85,7 +111,44 @@ public void setRole(String role) {
 
     @Override
     public String toString() {
-        return String.format("User {id= %s, password= %s, role= %s, locked = %s, attempts =%s}", id, password, role, locked, attempts);
+        return String.format("User {id= %s, name= %s, sureName= %s, email= %s, password= %s, role= %s, attempts =%s, locked = %s}",
```

Comment:
please, separate returning and actual method logic. it looks not well.... and surname is ONE word. please, change to "surename"